src/utils/FormSubmissionHelper.js

```diff
@@ -0,0 +1,65 @@
+import { Session } from '@inrupt/solid-client-authn-browser';
```

Comment:
Switching this Session with the Session import from `@inrupt/solid-ui-react` since it's part of the useSession hook for the SDK's SessionProvider.

Reply:
Fixed.